prql-compiler/src/lib.rs

```diff
@@ -128,6 +127,10 @@ pub fn compile(prql: &str, options: Option<sql::Options>) -> Result<String, Erro
         .map_err(|e| e.composed("", prql, false))
 }
 
+#[doc = include_str!("../README.md")]
```

Comment:
That's clever, I didn't know this was possible

Reply:
> > One thing I don't love about this is that change 2 duplicates the [rustdoc doctest](https://github.com/nkicg6/prql/blob/a980af2b72040c3fe99946d5530e781d24c385cb/prql-compiler/src/lib.rs#L130-L132) in change 3. I think it is useful to have that example in the rust docs page, and I couldn't figure out how to include the readme example and have it also show up in the rust docs, so I left it as is.

Ah, so it tests it, but it doesn't appear in the documentation itself? It's still much better even though it's duplicated.

Reply:
right! I like it too, I stumbled on it earlier today in the rustdoc docs. I think it fits this use case well by linking the readme, library, and book bindings to a test. Thanks for your edits and review I'll incorporate those shortly.